highfidelity/hifi
# MS13955: Audio Stats are blank

This should fix MS[#13955](https://highfidelity.fogbugz.com/f/cases/13955/Audio-Stats-are-blank).

**TEST PLAN**

- Launch Interface
- Open `GOTO` app in tablet/toolbar (in either HMD/Desktop mode)
- Go to a domain listed in the ones available
- Enable `Settings->Developer Menu`
- Open audio statistics via `Developer->Audio->Stats...`
- The popup window should display audio interface statistics
